### DISCUSSION TITLE
Server Error while try to open Scripts menu

### DISCUSSION THREAD

**Discussion by eudjinnl** [netbox-community/netbox#12111] [Q&A] [answered]
Hi all.

I face the server error trying to open Other - Scripts
I get a Server Error message:

Server Error
There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'TypeError'>

ObjectVar.__init__() missing 1 required positional argument: 'model'

Python version: 3.10.6
NetBox version: 3.4.7
If further assistance is required, please post to the [NetBox discussion forum](https://github.com/netbox-community/netbox/discussions) on GitHub.

I have 2 Netbox installations (v.3.3.1 and v3.4.7). Both of them have the same issue.

**Comment by candlerb** [comment, marked as answer]
Almost certainly it's something wrong with one of your custom scripts.

To get a backtrace, set ADMIN and EMAILS in configuration.py (and it will be E-mailed to you), or set DEBUG=True (and you'll get it in the web interface). This will tell you exactly which file is at fault.

The error suggests that there's a script where someone has put `ObjectVar()` instead of `ObjectVar(ModelClass)` - see [docs](https://docs.netbox.dev/en/stable/customization/custom-scripts/#objectvar).

Check all files under `/opt/netbox/netbox/scripts/` - although as I say, it's easier if you get a backtrace as it will tell you exactly which one.

**Reply by eudjinnl** [reply]
Check all files under /opt/netbox/netbox/scripts/

Should this folder be empty by default?

**Reply by candlerb** [reply]
I believe that in a fresh install, it should just have a zero-byte `__init__.py` file.

Any other `*.py` files in there are things you installed yourself.

**Reply by eudjinnl** [reply]
Thank you for your answers.
Switched on a debug and as you predicted got a faulty .py file in  /opt/netbox/netbox/scripts/
I recall that I tried to experument with 3d party plugin and it looks like I removed it incorrectly so the file remained in folder.
I've removed it and now has no errors.

Thank you once again
